core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -569,38 +543,14 @@ public void testMultipleViewsDifferentMeasures() {
     StatsTestUtil.assertAggregationMapEquals(
         viewData1.getAggregationMap(),
         ImmutableMap.of(
-            Arrays.asList(VALUE), StatsTestUtil.createAggregationData(AGGREGATIONS, 1.1)),
+            Arrays.asList(VALUE), StatsTestUtil.createAggregationData(MEAN, 1.1)),
         EPSILON);
     assertThat(viewData2.getWindowData())
         .isEqualTo(CumulativeData.create(Timestamp.create(2, 0), Timestamp.create(4, 0)));
     StatsTestUtil.assertAggregationMapEquals(
         viewData2.getAggregationMap(),
         ImmutableMap.of(
-            Arrays.asList(VALUE), StatsTestUtil.createAggregationData(AGGREGATIONS, 2.2)),
-        EPSILON);
-  }
-
-  @Test
-  public void testGetCumulativeViewDataWithoutBucketBoundaries() {
```

Comment:
I think we should keep this test for when we add back Distribution.

Reply:
Done, though currently `Distribution` needs to have non-null `BucketBoundaries`.

Reply:
Sorry, my comment wasn't very clear.  I meant that I think we should test Distribution in all of the other tests and use this test to test a different aggregation (e.g., Mean) after you add Distribution.  I think that leaving it commented out for now is fine, though.